src/data/finset.lean

```diff
@@ -1105,6 +1105,9 @@ theorem card_image_of_injective [decidable_eq β] {f : α → β} (s : finset α
   (H : function.injective f) : card (image f s) = card s :=
 card_image_of_inj_on $ λ x _ y _ h, H h
 
+@[simp] lemma card_map {α β} [decidable_eq β] {f : α ↪ β} {s : finset α} : (s.map f).card = s.card :=
```

Comment:
```suggestion
@[simp] lemma card_map {α β} [decidable_eq β] (f : α ↪ β) {s : finset α} : (s.map f).card = s.card :=
```
Since `f` does not occur on the right hand side of the equation, it should be an explicit variable. Otherwise it become unnecessary hard to `rw ←` with this lemma.